proto/cosmos/gov/query.proto

```diff
@@ -4,34 +4,51 @@ package cosmos.gov;
 import "cosmos/query/pagination.proto";
 import "gogoproto/gogo.proto";
 import "cosmos/gov/gov.proto";
+import "google/api/annotations.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types";
 
 // Query defines the gRPC querier service for gov module
 service Query {
   // Proposal queries proposal details based on ProposalID
-  rpc Proposal (QueryProposalRequest) returns (QueryProposalResponse) {}
+  rpc Proposal (QueryProposalRequest) returns (QueryProposalResponse) {
+    option (google.api.http).get = "/cosmos/gov/v1beta/proposal/{proposal_id}";
```

Comment:
I suggest to use plural everywhere
```suggestion
    option (google.api.http).get = "/cosmos/gov/v1beta/proposals/{proposal_id}";
```

Reply:
Having singular makes more sense to me, in cases where data being queried is of singular proposal.

Reply:
Hmm but I see most of the people uses plural and having a tree like structure looks better and official grpc gateway docs also following the same.

Reply:
Agreed with @anilCSE, I generally see plurals everywhere for REST API resources.